lib/util/mapObject.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+export interface MapObject<T> {
+  [name: string]: T
+}
+
+export function objectMap<T>(src: MapObject<T>, f: (v: T, name: string) => T): MapObject<T> {
+  const result: MapObject<T> = {}
+  for (const name in src) {
```

Comment:
>name in src) [](start = 13, length = 12)

is this the only reason we disabled forin rule ? cant we use Object.keys here  ?

Reply:
with `for of` on it I mean

---
In reply to: [197878939](https://github.com/Azure/oav/pull/265#discussion_r197878939) [](ancestors = 197878939)